.github/workflows/ci.yaml

```diff
@@ -1392,7 +1392,15 @@ jobs:
           shellcheck -x -s dash -- rustup-init.sh
           git ls-files -- '*.sh' | xargs shellcheck -x -s dash
           git ls-files -- '*.bash' | xargs shellcheck -x -s bash
-      - name: Run formatting checks
+      - name: Install taplo
+        uses: taiki-e/install-action@v2
+        with:
+          tool: taplo-cli@0.8
```

Comment:
The latest version is 0.9, but for now both Neovim/mason and VSCode are on 0.8, so I'll stick with that until later.

Reply:
... I guess it'd make more sense once the VSCode plugin has been updated to 0.9. This might take a while since they'll have to wait for the maintainer to rotate the access token to the marketplace (https://github.com/tamasfe/taplo/issues/624)... Marking as draft until this problem is resolved.